lib/cli/src/repro-generators/configs.ts

```diff
@@ -17,6 +17,10 @@ export interface Parameters {
   typescript?: boolean;
   /** Merge configurations to main.js before running the tests */
   mainOverrides?: Partial<StorybookConfig> & Record<string, any>;
+  /** Generator that downloads from remote repro in storybookjs/repro-templates on Github */
+  gitRepoGenerator?: string;
+  /** Whether to download from git templates */
+  preferGitTemplate?: boolean;
```

Comment:
I think this shouldn't be included in the code. It should just try & fail & run the full generator OR the user should pass `--skip-git`.

Reply:
That's the case. I just added that name to make it more apparent, but that flag is set based on --skip-git